src/Compilers/Core/Portable/Syntax/InternalSyntax/SyntaxNodeCache.cs

```diff
@@ -87,7 +87,9 @@ internal static void CacheHit()
             Console.WriteLine("RateOfCacheable: " + (cacheHits * 100 / (cacheableNodes)) + "%");
         }
 #else
+#pragma warning disable IDE0060 // Remove unused parameter
```

Comment:
:memo: see code in conditional preprocessor directive above

Reply:
@jcouv I guess you could use `_` as the name of the parameter when not used. It should "hopefully" suppress the warning. If it doesn't, then it's actually a bug.